packages/busybox/build.sh

```diff
@@ -32,17 +32,51 @@ termux_step_configure() {
 		-e "s|@TERMUX_CFLAGS@|$CFLAGS|g" \
 		-e "s|@TERMUX_LDFLAGS@|$LDFLAGS|g" \
 		-e "s|@TERMUX_LDLIBS@|log|g" \
-		$TERMUX_PKG_BUILDER_DIR/busybox.config > .config
+		"$TERMUX_PKG_BUILDER_DIR/busybox.config" > .config
 	unset CFLAGS LDFLAGS
 	make oldconfig
 }
 
 termux_step_make_install() {
 	# Using unstripped variant. The post-massage step will strip binaries anyway.
-	install -Dm700 ./0_lib/busybox_unstripped $TERMUX_PREFIX/bin/busybox
-	install -Dm700 ./0_lib/libbusybox.so.${TERMUX_PKG_VERSION}_unstripped $TERMUX_PREFIX/lib/libbusybox.so.${TERMUX_PKG_VERSION}
-	ln -sfr $TERMUX_PREFIX/lib/libbusybox.so.${TERMUX_PKG_VERSION} $TERMUX_PREFIX/lib/libbusybox.so
+	install -Dm700 "./0_lib/busybox_unstripped" "$TERMUX_PREFIX/bin/busybox"
+	install -Dm700 "./0_lib/libbusybox.so.${TERMUX_PKG_VERSION}_unstripped" "$TERMUX_PREFIX/lib/libbusybox.so.${TERMUX_PKG_VERSION}"
+	ln -sfr "$TERMUX_PREFIX/lib/libbusybox.so.${TERMUX_PKG_VERSION}" "$TERMUX_PREFIX/lib/libbusybox.so"
 
 	# Install busybox man page.
-	install -Dm600 -t $TERMUX_PREFIX/share/man/man1 $TERMUX_PKG_SRCDIR/docs/busybox.1
+	install -Dm600 -t "$TERMUX_PREFIX/share/man/man1" "$TERMUX_PKG_SRCDIR/docs/busybox.1"
+
+	mkdir -p "$TERMUX_PREFIX/libexec/busybox"
+	{ # Set up a wrapper script to let `busybox vi` provide `vi` or `editor`
+		echo "#!$TERMUX_PREFIX/bin/sh"
+		echo "exec busybox vi \"\$@\""
+	} > "$TERMUX_PREFIX/libexec/busybox/vi"
+	chmod 700 "$TERMUX_PREFIX/libexec/busybox/vi"
```

Comment:
I just noticed this section isn't necessary.
We can just call `busybox` as `vi`.
And then have `vi` as an `editor` alternative.